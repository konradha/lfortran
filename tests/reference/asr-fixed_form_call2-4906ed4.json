{
    "basename": "asr-fixed_form_call2-4906ed4",
    "cmd": "lfortran --show-asr --implicit-typing --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_call2.f",
    "infile_hash": "94899bb360f6a9205dc9fe3c76e28e8c04775b011747845b84950eb3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-fixed_form_call2-4906ed4.stderr",
    "stderr_hash": "bbfd1ed055ecdf95b387a4d3d4df4a1ac15e3663d54e242668662969",
    "returncode": 2
}