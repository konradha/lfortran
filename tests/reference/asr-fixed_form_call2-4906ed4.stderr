semantic error: Function 'prinf' not found or not implemented yet (if it is intrinsic)
 --> tests/fixed_form_call2.f:4:9
  |
4 |         call prinf('lw = *',1)
  |         ^^^^^^^^^^^^^^^^^^^^^^ 
